systemtest/src/test/java/io/strimzi/systemtest/upgrade/regular/StrimziDowngradeST.java

```diff
@@ -95,6 +100,10 @@ void setupEnvironment() {
 
     @AfterEach
     void afterEach() {
+        // delete all topics created in test
+        cmdKubeClient(TestConstants.CO_NAMESPACE).deleteAllByResource(KafkaTopic.RESOURCE_KIND);
+        KafkaTopicUtils.waitForTopicWithPrefixDeletion(TestConstants.CO_NAMESPACE, topicName);
```

Comment:
Just thinking...all CR topis are created by tests? If so, why ResourceManager doesn't handle them?

Reply:
ResourceManager deletes them, but after this AfterEach. Inside this one there is a method for deleting the applied files - even the CRDs. But because of finalizers, you cannot delete the KafkaTopic CRD, thus the process willbe stuck. Removing the KafkaTopics before that will ensure that everything will be cleared without a problem

Reply:
Ah sorry, I missed the "all CR topics" - most of them are deployed using resourceManager, but I guess one or two are deployed using YAML and the `cmdKubeClient`
So that's why there are extra cleaning steps (to remove YAML files applied during tests before ResourceManager takes care about other CRs)